app/views/users/index.html.erb

```diff
@@ -1,2 +1,13 @@
-<h1>Users#index</h1>
-<h2>This is the Users index page</h2>
\ No newline at end of file
+<div class="main-container">
+	<% @users.each do |user| %>
+		<div class="users-wrapper flex">
+			<div class="img-cont">
+				<%# <%= image_tag user.photo, :size => "50x50", alt: "user photo" %> %>
+			</div>
+			<div class="user-details">
+				<h2><%= user.name %></h2>
+				<p>Number of posts: <%= user.posts_counter %></p>
+			</div>
+		</div>
+	<%end%>
+</div>
```

Comment:
- [ ] [OPTIONAL] user profile picture is not showing, kindly fix this 
![Screenshot from 2023-06-17 00-17-52](https://github.com/Abe1able/Bloggingly/assets/66097371/ae8b3092-244f-4912-914c-a4102aac015c)

Reply:
Fixed, Thank you